custom_reports/integrations/google_search_console__dimensions.rst

```diff
@@ -3,11 +3,11 @@
     +---------------------+---------------------------+--------+----------+--------------+--------+--------------------------------------------------------------------------------------------------+
     |   Dimension Name    |         Column ID         | Scope  |   Type   |Database Type |Nullable|                                              Notes                                               |
     +=====================+===========================+========+==========+==============+========+==================================================================================================+
-    |Source               |source                     |session |str       |string        |False   |                                                                                                  |
+    |Source               |source                     |session |str_nocase|string        |False   |                                                                                                  |
     +---------------------+---------------------------+--------+----------+--------------+--------+--------------------------------------------------------------------------------------------------+
-    |Medium               |medium                     |session |str       |string        |False   |                                                                                                  |
+    |Medium               |medium                     |session |str_nocase|string        |False   |                                                                                                  |
     +---------------------+---------------------------+--------+----------+--------------+--------+--------------------------------------------------------------------------------------------------+
-    |Source/Medium        |source_medium              |session |str       |string        |False   |                                                                                                  |
+    |Source/Medium        |source_medium              |session |str_nocase|string        |False   |                                                                                                  |
```

Comment:
`str_nocase` is only Our internal type? I don't find any detailed information in our docs about that (our docs is first search results) and it don't look as standard widely used 

https://www.google.com/search?q=str_nocase

Reply:
yeah, we should probably add some description one day - this `str_nocase` is already in [docs](https://developers.piwik.pro/en/latest/custom_reports/columns.html#:~:text=session_ecommerce_status.json-,Source,-source) so I just updated gsc/gads docs